src/vivarium_gates_nutrition_optimization/model_specifications/model_spec.yaml

```diff
@@ -7,6 +7,7 @@ components:
 #            - DisabilityObserver()
             - MortalityObserver()
             - ResultsStratifier()
+            - DiseaseObserver()
```

Comment:
I would have expected you need the name of the disease component as an arg for `DiseaseObserver`, e.g., `DiseaseObserver("ischemic_stroke")` or `DiseaseObserver("pregnancy")` for this usage.

Reply:
arg, you are right

Reply:
I did that in implementation as I was testing and forgot to add it back here

Reply:
fixed